src/AppBundle/Controller/PasswordController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace AppBundle\Controller;
+
+use eZ\Publish\API\Repository\UserService as UserServiceInterface;
+use EzSystems\EzPlatformUser\Form\Data\UserPasswordChangeData;
+use EzSystems\EzPlatformUser\Form\Type\UserPasswordChangeType;
+use Psr\Log\LoggerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class PasswordController
```

Comment:
@SylvainGuittard / @lserwatka  Shouldn't this be in user package so developers can reuse this?
ref: https://github.com/ezsystems/ezplatform-user

Reply:
Yes, but we need to change code in the ezplatfrom-user. This is stage 2. Stage 1 was to extract code into the separate bundle.

Reply:
Actually, this controller might not be needed here at all. @damianz5 is checking. Entire logic (exactly the same) in the ezplatfrom-user bundle.